troposphere/static/js/stores/ProjectStore.js

```diff
@@ -1,11 +1,23 @@
 import Dispatcher from "dispatchers/Dispatcher";
 import BaseStore from "stores/BaseStore";
+import Project from "models/Project";
 import ProjectCollection from "collections/ProjectCollection";
 import ProjectConstants from "constants/ProjectConstants";
 
 let ProjectStore = BaseStore.extend({
     collection: ProjectCollection,
 
+    getSharedProject: function() {
+        var sharedProject = new Project({
+            id: 'shared',
+            owner: 'shared',
+            name: 'My Shared Resources',
+            description: "This project contains shared resources. To share a resource with another user, create an instance and select 'Share Access'. After approving the request, your resources will be found here.",
+            leaders: [],
+            users: [],
+        });
+        return sharedProject;
+    },
```

Comment:
Forgot to point this out in the review:

Here is a way to use the existing system to create a "namespace" for the shared project space/URL route.

By using `shared` as the `UUID` for the project, I can now create (and possibly advertise in-app?) a share link that works for all users who i've shared a specific instance with to immediately see and access my instance:
`https://mickey.cyverse.org/application/projects/shared/instances/36023`